src/control/traversable/basic.lean

```diff
@@ -59,11 +60,12 @@ section applicative_transformation
 variables (F : Type u → Type v) [applicative F] [is_lawful_applicative F]
 variables (G : Type u → Type w) [applicative G] [is_lawful_applicative G]
 
-/-- A transformation between applicative functors.  It is similar to a
-natural transformation, but instead the function `app` preserves the
-`pure` and `seq` (`<*>`) operations. -/
+/-- A transformation between applicative functors.  It a natural
+transformation such that `app` preserves the `has_pure.pure` and
+`functor.map` (`<*>`) operations. See
+`applicative_transformation.preserves_map` for naturality. -/
 structure applicative_transformation : Type (max (u+1) v w) :=
-(app : ∀ α : Type u, F α → G α)
```

Comment:
There's lots of unrelated changes in this PR.  Does it depend on some other PRs?

Reply:
This should be fixed now